app/src/main/java/tech/pauly/findapet/discover/AnimalListItemViewModel.java

```diff
@@ -50,4 +68,19 @@ private void setPhoto(Media media) {
             }
         }
     }
+
+    public static class Factory {
+        private ViewEventBus eventBus;
+        private TransientDataStore dataStore;
+
+        @Inject
+        public Factory(ViewEventBus eventBus, TransientDataStore dataStore) {
+            this.eventBus = eventBus;
+            this.dataStore = dataStore;
+        }
+
+        public AnimalListItemViewModel newInstance(Animal animal) {
```

Comment:
First use case of using a Factory in this project. My go-to solution for when you need both 
1. An object made with custom data sent into it (Animal)
2. That object instantiated with objects from the object graph

This doesn't work when the objects being injected need to have one instance per class being constructed via the factory (ie only Singletons) but many dependencies are Singletons so it's good for now.